plotly_resampler/figure_resampler/figure_resampler.py

```diff
@@ -238,3 +240,6 @@ def register_update_graph_callback(
             dash.dependencies.Input(graph_id, "relayoutData"),
             prevent_initial_call=True,
         )(self.construct_update_data)
+
+    def _ipython_display_(self):
+        return self.show_dash(mode="inline")
```

Comment:
Hmmm, users should be able to control the `show_dash` kwargs (e.g., the port).

One solution is to log these in the constructor of `FigureResampler`?

Reply:
When registering, these kwargs can be passed as `constr_kwargs`  to `register_plotly_resampler`